lib/models/revision.js

```diff
@@ -14,7 +15,7 @@ var dmpWorker = createDmpWorker()
 var dmpCallbackCache = {}
 
 function createDmpWorker () {
-  var worker = childProcess.fork('./lib/workers/dmpWorker.js', {
+  var worker = childProcess.fork(path.join(__dirname, '../workers/dmpWorker.js'), {
```

Comment:
Can we use `path.resolve()` here as well? I see no point in not using resolve everywhere.